client/a8c-for-agencies/components/offering/style.scss

```diff
@@ -0,0 +1,92 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+
+.a4a-offering-card__wrapper {
+	padding-top: 16px;
```

Comment:
[Non-blocker] Can we use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) everywhere on this file since we can support RTL locales in Calypso.

Reply:
Sure, thank you! Sorry, I'm still not used for these.